### ISSUE TITLE
Keybindings don't work with PDF viewer

### ISSUE THREAD

**Issue by robsmith11** [cmcaine/tridactyl#107]
I imagine this is a current webextension limitation like #102, but I'm adding this because I don't see an issue for it yet and I read a lot of PDFs in firefox.

**Comment by bovine3dom** [comment]
You could just use an external viewer such as Okular or Sumatra PDF, both of which have some vim-style binds. FWIW, the pdf.js reader built into Firefox is pretty awful at font rendering etc...

**Comment by bovine3dom** [comment]
@robsmith11 I'm actually finding that most keys work in the PDF reader in the browser; it seems to only be scrolling that is a problem. Could you confirm this?

This is therefore a likely duplicate of #35 or #122.

**Comment by cmcaine** [comment]
I forgot I was porting the scrolling logic from vimperator. We should merge that :P

**Comment by robsmith11** [comment]
Yes, I suppose it is just scrolling.  Feel free to close this if appropriate.

**Comment by augustebaum** [comment]
Hi, in my case scrolling with `j`/`k` does work (although it scrolls one full page which is less than ideal -- is there a setting I can change for this?), but more importantly the keybindings to change tabs no longer work; PDF pages are like black holes I can never get out of. Is this behaviour documented anywhere? Thanks very much and thanks also for tridactyl in general!

**Comment by bovine3dom** [comment]
You can press ctrl+, to escape from any page/part of Firefox that you are trapped in back to somewhere where you can use Tridactyl :)

Otherwise you need to look at #541 but it isn't good news - Mozilla stopped us from running on PDF pages. We could bundle our own PDF viewer to work around it but I haven't got around to doing it.

**Comment by augustebaum** [comment]
Thanks for the useful shortcut! This is disheartening, but I guess I can always use my native PDF viewer. Thanks again!
